### PR TITLE
fix(DB/quest): 9175 - Fixing typo

### PR DESCRIPTION
**Pull request by wowquester**
## Changes Proposed:
-  Added blank space between sentences - quest_offer_reward
-  

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/16691

## SOURCE:

https://www.wowhead.com/wotlk/quest=9175/the-ladys-necklace

## Tests Performed:

- Tested myself, works

## How to Test the Changes:

1. horde char
2. .additem 22597
3. start quest
4. .go xyz 7552.79998 -6760.879883 91.337097
## Known Issues and TODO List:

- [No, but since this is my first PR, please check if everything is ok, i'm slightly nervous]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: data/sql/updates/pending_db_world/fix-quest-9175-typo.sql [hunk 1]
```diff
@@ -0,0 +1 @@
+UPDATE `quest_offer_reward` SET `RewardText` = 'You say that you found it on one of the Scourge at Windrunner Spire and that there\'s an inscription on it? Let me see! Here, take this coin for a job well done. ' WHERE (`ID` = 9175);
```
Comment (Helias) [inline data/sql/updates/pending_db_world/fix-quest-9175-typo.sql:1]: You should add two new line after the `Let me see!`

Comment (Helias) [inline data/sql/updates/pending_db_world/fix-quest-9175-typo.sql:1]: This should make the two newline
```suggestion
UPDATE `quest_offer_reward` SET `RewardText` = 'You say that you found it on one of the Scourge at Windrunner Spire and that there\'s an inscription on it? Let me see! $B$B Here, take this coin for a job well done. ' WHERE (`ID` = 9175);
```

**Review by Helias — APPROVED** [review]

**Review by Gultask — CHANGES_REQUESTED** [review]
ditto, needs sniff: https://www.azerothcore.org/wiki/sniffing-and-parsing

**Comment by Gultask** [comment]
Ah, sorry I didn't notice that it had another open that did the same thing: https://github.com/azerothcore/azerothcore-wotlk/pull/16686
Thanks for the PR, though!

**Comment by wowquester** [comment]
> Ah, sorry I didn't notice that it had another open that did the same thing: #16686 Thanks for the PR, though!

Gultask, https://github.com/azerothcore/azerothcore-wotlk/pull/16686 is for 1945, this one is for 1975.

**Comment by Gultask** [comment]
Oh pft. I'm sorry, lol.

Thankfully I have sniffs on hand so I'll check in a bit

**Comment by Gultask** [comment]
```
QuestGiverCreatureID: 16252
QuestID: 9175
[0] QuestFlags: 0
[1] QuestFlags: 0
[2] QuestFlags: 0
SuggestedPartyMembers: 0
EmotesCount: 1
Type: 6
Delay: 0
AutoLaunched: False
Unused: False
(QuestRewards) ChoiceItemCount: 0
(QuestRewards) ItemCount: 0
(QuestRewards) [0] ID: 0
(QuestRewards) [0] Quantity: 0
(QuestRewards) [1] ID: 0
(QuestRewards) [1] Quantity: 0
(QuestRewards) [2] ID: 0
(QuestRewards) [2] Quantity: 0
(QuestRewards) [3] ID: 0
(QuestRewards) [3] Quantity: 0
(QuestRewards) RewardMoney: 158500
(QuestRewards) XP: 3450
(QuestRewards) ArtifactXP: 0
(QuestRewards) ArtifactCategoryID: 0
(QuestRewards) Honor: 0
(QuestRewards) Title: 0
(QuestRewards) FactionFlags: 0
(QuestRewards) [0] FactionID: 922
(QuestRewards) [0] FactionValue: 5
(QuestRewards) [0] FactionOverride: 100000
(QuestRewards) [0] FactionCapIn: 7
(QuestRewards) [1] FactionID: 0
(QuestRewards) [1] FactionValue: 0
(QuestRewards) [1] FactionOverride: 0
(QuestRewards) [1] FactionCapIn: 7
(QuestRewards) [2] FactionID: 0
(QuestRewards) [2] FactionValue: 0
(QuestRewards) [2] FactionOverride: 0
(QuestRewards) [2] FactionCapIn: 7
(QuestRewards) [3] FactionID: 0
(QuestRewards) [3] FactionValue: 0
(QuestRewards) [3] FactionOverride: 0
(QuestRewards) [3] FactionCapIn: 7
(QuestRewards) [4] FactionID: 0
(QuestRewards) [4] FactionValue: 0
(QuestRewards) [4] FactionOverride: 0
(QuestRewards) [4] FactionCapIn: 7
(QuestRewards) [0] SpellCompletionDisplayID: 0
(QuestRewards) [1] SpellCompletionDisplayID: 0
(QuestRewards) [2] SpellCompletionDisplayID: 0
(QuestRewards) SpellCompletionID: 0
(QuestRewards) [0] CurrencyID: 0
(QuestRewards) [0] CurrencyQty: 0
(QuestRewards) [1] CurrencyID: 0
(QuestRewards) [1] CurrencyQty: 0
(QuestRewards) [2] CurrencyID: 0
(QuestRewards) [2] CurrencyQty: 0
(QuestRewards) [3] CurrencyID: 0
(QuestRewards) [3] CurrencyQty: 0
(QuestRewards) SkillLineID: 0
(QuestRewards) NumSkillUps: 0
(QuestRewards) TreasurePickerID: 0
(QuestRewards) (ItemChoiceData) [0] LootItemType: 0 (Item)
(QuestRewards) (ItemChoiceData) [0] ItemID: 0 (0)
(QuestRewards) (ItemChoiceData) [0] HasItemBonus: False
(QuestRewards) (ItemChoiceData) [0] Quantity: 0
(QuestRewards) (ItemChoiceData) [1] LootItemType: 0 (Item)
(QuestRewards) (ItemChoiceData) [1] ItemID: 0 (0)
(QuestRewards) (ItemChoiceData) [1] HasItemBonus: False
(QuestRewards) (ItemChoiceData) [1] Quantity: 0
(QuestRewards) (ItemChoiceData) [2] LootItemType: 0 (Item)
(QuestRewards) (ItemChoiceData) [2] ItemID: 0 (0)
(QuestRewards) (ItemChoiceData) [2] HasItemBonus: False
(QuestRewards) (ItemChoiceData) [2] Quantity: 0
(QuestRewards) (ItemChoiceData) [3] LootItemType: 0 (Item)
(QuestRewards) (ItemChoiceData) [3] ItemID: 0 (0)
(QuestRewards) (ItemChoiceData) [3] HasItemBonus: False
(QuestRewards) (ItemChoiceData) [3] Quantity: 0
(QuestRewards) (ItemChoiceData) [4] LootItemType: 0 (Item)
(QuestRewards) (ItemChoiceData) [4] ItemID: 0 (0)
(QuestRewards) (ItemChoiceData) [4] HasItemBonus: False
(QuestRewards) (ItemChoiceData) [4] Quantity: 0
(QuestRewards) (ItemChoiceData) [5] LootItemType: 0 (Item)
(QuestRewards) (ItemChoiceData) [5] ItemID: 0 (0)
(QuestRewards) (ItemChoiceData) [5] HasItemBonus: False
(QuestRewards) (ItemChoiceData) [5] Quantity: 0
(QuestRewards) IsBoostSpell: False
QuestPackageID: 0
PortraitGiver: 0
PortraitGiverMount: 0
PortraitGiverModelSceneID: 0
PortraitTurnIn: 0
QuestGiverCreatureID: 16252
QuestTitle: The Lady's Necklace
RewardText: You say that you found it on one of the Scourge at Windrunner Spire and that there's an inscription on it? Let me see!$B$BHere, take this coin for a job well done.
PortraitGiverText: 
PortraitGiverName: 
PortraitTurnInText: 
PortraitTurnInName: 
```
Modified the file to match sniff

**Review by Gultask — APPROVED** [review]

**Comment by Gultask** [comment]
Ty for the contribution!

**Pull request merged**